os/provisioning.md

```diff
@@ -30,25 +32,27 @@ etcd:
   discovery:                   https://discovery.etcd.io/<token>
 ```
 
-But what are `PUBLIC_IPV4` and `PRIVATE_IPV4`? These variables are populated from the environment in which Container Linux runs (e.g. EC2, Azure, GCE), assuming this metadata exists. In the default EC2 case, the `public_ipv4` and `local_ipv4` metadata would be used. On bare metal, this information cannot be reliably derived in a general manner, so these variables couldn't be used. In Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. These are just a few examples. Given the many different environments in which Container Linux can run, it's difficult if not impossible to accurately determine these variables in every scenario.
+`PUBLIC_IPV4` and `PRIVATE_IPV4` are automatically populated from the environment (such as EC2, Azure, or GCE) in which Container Linux runs, if this metadata exists. For example: For EC2 (the default), the `public_ipv4` and `local_ipv4` metadata would be used. In Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. On bare metal, this information cannot be reliably derived in a general manner, so these variables cannot be used.
```

Comment:
EC2 isn't the default. That statement is referring to the bare-bones config we give as an example in the EC2 docs (default is probably not the right word).